pkg/datapath/linux/node.go

```diff
@@ -727,7 +726,8 @@ func (n *linuxNodeHandler) insertNeighbor(ctx context.Context, newNode *nodeType
 	if nextHopIsNew || refresh {
 		hwAddr, _, err := arping.PingOverIface(nextHopIPv4, n.neighDiscoveryLink, srcIPv4)
 		if err != nil {
-			scopedLog.WithError(err).Error("arping failed")
+			scopedLog.WithError(err).Info("arping failed")
+			metrics.ArpingRequestsTotal.WithLabelValues(failed).Inc()
```

Comment:
@brb looks like this drags in 42eb1edafdaccad2814e7c16a1795e501fcd76b3 as well, which does not apply cleanly, I suppose I should just drop this line from the backport?

Reply:
Also adding metrics is actually feature, albeit a small one...

Reply:
The upstream commit https://github.com/cilium/cilium/commit/2de65d0e77df7d958ace1839c595261877e4988d does not include the metric change. Did it get by accident?

Reply:
Ah, I see, it must have slipped when I was resolving the conflict...